### PR TITLE
Responsive design

### PR DESCRIPTION
**Pull request by fededirocco**
## Summary
* Implemented responsive design to website
* Added Media Query to Navbar, Tech container and Footer scss.
* Added breakpoint at 1024px

## Trello Card
https://trello.com/c/YjHGoBwO/12-redise%C3%B1o-de-la-p%C3%A1gina

## Screenshots
image
image
image
image

### CODE REVIEW

File: scss/tech.scss [hunk 1]
```diff
@@ -45,3 +45,29 @@
...
+
+    .first {
+      border-right: none;
+    }
+    
+    .box {
+      border-bottom: solid $green 3px;
+    }
+
+    .last {
```
Comment (jjuli93) [inline scss/tech.scss]: you can use the last child pseudo-class. https://developer.mozilla.org/en-US/docs/Web/CSS/:last-child

File: scss/footer.scss [hunk 2]
```diff
@@ -23,3 +23,21 @@ footer {
...
+
+@media only screen and (max-width: 1024px) {
+  footer {
+    .changers-text {
+      font-size: 20px;
+      padding-top: 30px;
+    }
+
+    .logo {
+      width: 300px;
```
Comment (FrankIglesias) [inline scss/footer.scss]: use width and max-width outside the logo class to avoid this

File: scss/tech.scss [hunk 3]
```diff
@@ -56,3 +56,25 @@
     }
   }
 }
+
+@media only screen and (max-width: 1024px) {
+  .tech-grid-container {
```
Comment (FrankIglesias) [inline scss/tech.scss]: use grid template columns here

File: scss/footer.scss [hunk 4]
```diff
@@ -23,3 +23,22 @@ footer {
     font-size: 10px;
   }
 }
+
+@media only screen and (max-width: 1024px) {
+  footer {
```
Comment (FrankIglesias) [inline scss/footer.scss]: remove tags from scss

File: scss/tech.scss [hunk 5]
```diff
@@ -37,3 +37,23 @@
...
+    border-right: none;
+    grid-template-columns: auto;
+    grid-template-rows: repeat(5, auto);
```
Comment (FrankIglesias) [inline scss/tech.scss]: I think you can avoid this

**Pull request merged**